libs/platform/src/lib/components/form/form-options.ts

```diff
@@ -7,3 +7,17 @@ export type FormZone = 'zTop' | 'zBottom' | 'zLeft' | 'zRight';
 export type Column = 1 | 2 | 3 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
 
 export type Status = 'success' | 'error' | 'warning' | 'default' | 'information';
+
+export interface ColumnLayout {
+    XL: number;
+    L: number;
+    M: number;
+    S: number;
```

Comment:
Maybe it's a good idea to simplify the object for end-user and inherit previous value if none was provided?
I mean if user provides value for S size, we could automatically map it to the other sizes like M,L,XL.

Reply:
fixed